plugins/woocommerce/tests/php/src/Templates/Checkout_Review_Order_Hook_Test.php

```
@@ -0,0 +1,198 @@
+		// Clean up
+		WC_Helper_Product::delete_product( $product2->get_id() );
+	}
+
+	/**
+	 * @testdox Hook can be used to output HTML content.
+	 */
+	public function test_hook_can_output_html() {
+		$test_output = '<div class="test-hook-output">Test Content</div>';
+
+		// Add a callback that outputs HTML
+		add_action( 'woocommerce_review_order_after_cart_item_meta', function( $product, $cart_item, $cart_item_key ) use ( $test_output ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$product'. (undefined)

(UnusedFormalParameter)